draft-ietf-rats-architecture.md

```diff
@@ -212,6 +212,9 @@ Then the Claims Collector combines the Attestation Results into
 the final Evidence of the whole Composite Attester which is sent off to the remote
 Verifier, which might treat the claims obtained from the local Attestation Results
 as if they were Evidence.
+Before receiving the Endorsements and Appraisal Policies for other Attesters,
+to increase the security, the Lead Attester may first generate Evidence about
```

Comment:
This sentence is confusing.  How exactly does it "increase the security" to do things in that order?   And if the other Attesters push their evidence to the lead attester, then the order can't be guaranteed anyway, right?

Reply:
In the normal Attester case, the Endorser and Verifier Owner needs to trust the Verifier before sending Endorsement and Appraisal Policy to the Verifier. So it is the same in the case that the Lead Attester has an internal Verifier. "increase the security" may not be appropriate, what I want to express is to establish the trust.

The Lead Attester collecting Evidence of other Attesters happens inside the Composite Attester, so I think it doesn't matter when to collect the Evidence.